app/controllers/application_controller.rb

```diff
@@ -1,2 +1,37 @@
 class ApplicationController < ActionController::API
+  class RequiredParams < StandardError; end
+  
+  rescue_from 'ActiveRecord::RecordNotFound' do |exception|
+    render json: { error: exception }, status: 403
+  end
+
+  # The follow section of the ApplicationController is for checking
```

Comment:
@cchanningallen added a params check to make sure we have intelligent responses if a required param is missing.  Output looks like this:

```
{"error":"The following param(s) are required but were not found: user_id"}